server.py

```diff
@@ -1,36 +1,67 @@
 """
-Inverntory Management System Server
+Inverntory Management System Service
+
+Paths:
```

Comment:
I guess it's better to put these in the readme file or the help information returned by the root call?

Reply:
Moved to README.md
Since you are in charge of the root call, you can modify it if you want.